### ISSUE TITLE
Viewer facelift - Issue details tab (sequences)

### ISSUE THREAD

**Issue by sanmont3drepo** [3drepo/3drepo.io#3173]
### Description
For the issues we need to style every tab. This one is for sequences

### Goals
- [ ] As a user I want to change issues sequences properties and have it styled in the v5 style

### Tasks
- [ ] Style the actual date picker (consult with George before starting this)
- [ ] Style blank state
- [ ] Style the input box for the picker  

### Notes:
We are not changing the button icons for now

### Related Resources
[MockUp](https://www.figma.com/file/J5I3hYQTrLCjEVP1iqcyZK/3D-Viewer?node-id=136544%3A177468)

**Comment by carmenfan** [comment]
Merged into #3168
